perun-core/src/main/java/cz/metacentrum/perun/core/entry/VosManagerEntry.java

```diff
@@ -380,6 +380,7 @@ public void removeAdmin(PerunSession sess, Vo vo, Group group) throws PrivilegeE
 		//Role can be only supported one (TopGroupCreator, VoAdmin or VoObserver)
```

Comment:
There should be mentioned the new supported role too.

Reply:
Done.